DatadogSDKBridge/Classes/Implementation/DdSdkImplementation.swift

```diff
@@ -57,14 +57,18 @@ internal class DdSdkImplementation: DdSdk {
         }
 
         switch configuration.site?.lowercased ?? "us" {
-        case "us":
-            _ = ddConfigBuilder.set(endpoint: .us)
-        case "eu":
-            _ = ddConfigBuilder.set(endpoint: .eu)
-        case "gov":
-            _ = ddConfigBuilder.set(endpoint: .gov)
+        case "us1", "us":
+            _ = ddConfigBuilder.set(endpoint: .us1)
+        case "eu1", "eu":
+            _ = ddConfigBuilder.set(endpoint: .eu1)
+        case "us3":
+            _ = ddConfigBuilder.set(endpoint: .us3)
+        case "us5":
+            _ = ddConfigBuilder.set(endpoint: .us5)
+        case "us1_fed", "gov":
+            _ = ddConfigBuilder.set(endpoint: .us1_fed)
```

Comment:
where do we document these? how can a new user know about those options?

Reply:
Ah good catch, it should be documented [here](https://github.com/DataDog/dd-bridge-ios/blob/main/DatadogSDKBridge/Classes/Interface/DdSdkConfiguration.swift#L17)!